utils/gyb.py

```diff
@@ -1079,6 +1079,20 @@ def execute_template(
 
 
 def main():
+    """
+    Lint this file.
+    >>> import subprocess, sys
+    >>> script_path = os.path.realpath(__file__)
+    >>> utils_directory = os.path.dirname(script_path)
+    >>> linting = subprocess.Popen([sys.executable,
+    ...                             os.path.join(utils_directory,
+    ...                                         'python-lint'),
+    ...                             script_path])
```

Comment:
Seems to me one shouldn't have to launch a separate python process just to invoke the linter, since... it's python?